### ISSUE TITLE
afFieldValueIs not working with Semantic UI checkbox,slider,toggle field

### ISSUE THREAD

**Issue by eoliphan** [fabienb4/meteor-autoform-semantic-ui#46]
Hi I'm having an issue that appears to be similar to this one: https://github.com/fabienb4/meteor-autoform-semantic-ui/issues/42

```
 </div>
        {{> afQuickField name="flip"  checkboxType="toggle"}}

        <hr/>
        {{#if afFieldValueIs name="flip" value="true"}}
        ....
```

After checking the 'flip' field, the html, etc in the {{#if afFieldValueIs }} block is not being displayed.  I tried the trick of deleting the .custom.semantic.json but that doesn't seem to have fixed the problem for me.

**Comment by fabienb4** [comment]
By default `Autoform.getFormValue` casts the returned value according to the schema's value type (see [autoform-api.js#L283](https://github.com/aldeed/meteor-autoform/blob/devel/autoform-api.js#L283)).

`afFieldValueIs` uses `Autoform.getFormValue` to retrieve the current value of the field, but it does not use the schema to determine the type of `value` specified in its parameters (see [autoform-helpers.js#L71](https://github.com/aldeed/meteor-autoform/blob/devel/autoform-helpers.js#L71)).

So basically your above code is like doing this:

``` js
if (true === "true")
```

Instead you will want to change your code to this:

```
{{#if afFieldValueIs name="flip" value=true}}
```

Autoform and its quirks....

**Comment by eoliphan** [comment]
Ah.. kicking myself :)  I actually had the 'bare' true initially, had tried switching it to a string just to see if it would work, but forgot to switch it back after I deleted .custom.semantic.json.  Working fine now
